pkg/engine/jmespath/arithmetic.go

```diff
@@ -16,6 +16,7 @@ type operand interface {
 	Multiply(interface{}) (interface{}, error)
 	Divide(interface{}) (interface{}, error)
 	Modulo(interface{}) (interface{}, error)
+	Round(interface{}) (interface{}, error)
```

Comment:
shouldn't the argument be an int here ?

Reply:
You're right but I have already whether the second argument is int or not.